stock_reserve_rule/readme/DESCRIPTION.rst

```diff
@@ -21,6 +21,9 @@ The included advanced removal strategies are:
 * Full Packaging: tries to remove full packaging (configured on the products)
   first, by largest to smallest package or based on a pre-selected package
   (default removal strategy is then applied for equal quantities).
+* By lot: tries to remove a lot. A specific field allows to set whether to reserve
+  a lot with qty matching requested qty, or a lot with qty >= requested quantity,
+  or only lots with qty > requested qty.
```

Comment:
* Single lot: tries to remove a single lot. This strategy requests to set a tolerance (either in percentage or absolute value) to allow selection of a lot with same qty as requested qty by setting a tolerance value of 0. 
By setting a Tolerance value different from 0, user can create rules to select a lot with qty higher than requested (Tolerance on = Upper Limit) or lower than requested (Tolerance on = Lower Limit). Note that rules with Tolerance value != 0 will not select lots with requested qty = lot qty.